.github/workflows/foxpuppet_deploy.yml

```diff
@@ -1,9 +1,9 @@
 name: Publish to PyPI
 
 on:
-  pull_request:
-    branches:
-      - main
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
Check these docs for more specifics https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags

Reply:
We probably want something like `'v[0-9]+.[0-9]+.[0-9]+'`